### DISCUSSION TITLE
Multiple persists in a store all save as false

### DISCUSSION THREAD

**Discussion by robdekort** [alpinejs/alpine#2350] [7. Bugs (archived)] [answered]
Possibly related to: #2317

When you have a store and want to persist multiple values:

```js
Alpine.store('myStore', {
    firstThing: Alpine.$persist('Hi'),
    secondThing: Alpine.$persist('is this'),
    thirdThing: Alpine.$persist('a bug?'),
    trueThing: Alpine.$persist(true),
    falseThing: Alpine.$persist(false),
})

```

They all save as `false` for me.

<img width="630" alt="Screenshot 2021-11-10 at 12 22 58" src="https://user-images.githubusercontent.com/69107412/141104499-2e91d2e3-3446-42ea-a897-10bf22048906.png">

**Comment by SimoTod** [comment, marked as answer]
Yep, related. The same PR will fix it (https://codepen.io/SimoTod/pen/wvqXVbx). It has been merged but not released yet.

**Reply by robdekort** [reply]
Brilliant!

**Reply by robdekort** [reply]
I guess we have it neatly documented now :-)
